staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -436,3 +451,18 @@ func (o *DeleteOptions) PrintObj(info *resource.Info) {
 	// understandable output by default
 	fmt.Fprintf(o.Out, "%s \"%s\" %s\n", kindString, info.Name, operation)
 }
+
+func (o *DeleteOptions) confirmation(infos []*resource.Info) bool {
+	fmt.Fprintf(o.Out, i18n.T("You are about to delete the following %d resource(s):\n"), len(infos))
```

Comment:
I'd mimic the message on that other comment of mine, that with the sample `rm` output. So maybe something like: `Remove pod/xyz (A/y/n)?`

Reply:
Actually that option `A` could only be used when `--all` or `--selector` was explicitly specified, in all other cases `y/n` should be sufficient.

Reply:
Since we decided to list all resources prior to delete(at least for now) and only expect y/n, `Remove pod/xyz (A/y/n)?` message seems not covering our case?